### PR TITLE
WW-342: Add flow tracking after VE is loaded

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
@Wikia/west-wing

### CODE REVIEW

File: extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require(['wikia.flowTracking.createPage', 'mw', 'jquery'], function (flowTrackingCreatePage, mw, $) {
+	function init() {
+		mw.hook('ve.activationComplete').add(function () {
+			flowTrackingCreatePage.trackOnEditPageLoad('ve');
```
Comment (rybmat) [inline extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js]: As we are using 'sourceedit', 'rte-source' and 'rte-wysiwyg' I'm wondering if we should use 've' or maybe 'visualeditor'. What do you think?

**Comment by rybmat** [comment]
one minor suggestion, rest LGTM

File: extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js (hunk 1 above)
Comment (lukaszkonieczny) [reply extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js]: I don't have strong opinion as long as we are able to distinguish editors

**Review by rybmat — APPROVED** [review]

File: extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js (hunk 1 above)
Comment (sebastianmarzjan) [reply extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js]: "visualeditor" is used in current tagging, so I'd use it

**Comment by sebastianmarzjan** [comment]
Please adjust the string literal and then GTG

**Review by sebastianmarzjan — APPROVED** [review]

**Pull request merged**